force-app/main/default/lwc/geGatewaySelectWidget/geGatewaySelectWidget.js

```diff
@@ -21,18 +21,24 @@ import GeGatewaySettings from 'c/geGatewaySettings';
 import { fireEvent } from 'c/pubsubNoPageRef';
 import { isEmpty, showToast } from 'c/utilCommon';
 
+const TEMPLATE_CSS_CLASS = 'template-top';
+const GIFT_ENTRY_CSS_CLASS = 'gift-entry-top';
+const TOKENIZE_ID = 'TOKENIZE';
+
 export default class GeGatewaySelectWidget extends LightningElement {
     @track isExpanded = false;
     @track selectedGateway = null;
     @track gatewayOptions = [];
-    @track preGatewayOptions = [];
     @track isACHEnabled = true;
     @track isACHDisabled = false;
     @track isCreditCardEnabled = true;
     @track isCreditCardDisabled = false;
     @track isLoading = true;
     @track isDefaultTemplate = false;
     @track isGatewayAssignmentEnabled = false;
+    @track isGatewaySelectionDisabled = false;
+
+    @api parentId = '';
```

Comment:
Should this be set to blank if it's being passed in from a parent component? I don't think it needs to be initialized at all.